### PR TITLE
Pass optimizer_nestloop_factor to DXL through optimizer config

### PR DESCRIPTION
**Pull request by sambitesh**
Earlier we were accounting for `optimizer_nestloop_factor` while translating
back from DXL in GPDB side in COptTasks::SetCostModelParams. Now it is being
taken care in GPORCA side.

Signed-off-by: Bhuvnesh Chaudhary <bchaudhary@pivotal.io>

### CODE REVIEW

**Comment by vraghavan78** [comment]
GPORCA PR: https://github.com/greenplum-db/gporca/pull/343

File: src/backend/gpopt/utils/COptTasks.cpp [hunk 1]
```diff
@@ -862,22 +864,6 @@ COptTasks::SetCostModelParams
 {
 	GPOS_ASSERT(NULL != pcm);
 
-	if (optimizer_nestloop_factor > 1.0)
```
Comment (vraghavan78) [inline src/backend/gpopt/utils/COptTasks.cpp]: can we set optimizer_nestloop_factor to 0 or less?

Comment (sambitesh) [reply src/backend/gpopt/utils/COptTasks.cpp]: It's lowest value is 1.0, from code

```
&optimizer_nestloop_factor,
		1024.0, 1.0, DBL_MAX, NULL, NULL
```

Comment (vraghavan78) [reply src/backend/gpopt/utils/COptTasks.cpp]: thanks for checking.

**Comment by dhanashreek89** [comment]
@sambitesh please see if this needs to be closed or updated with the new approach for passing the nestloop factor to ORCA

**Comment by sambitesh** [comment]
@dhanashreek89 We will raise a new PR. Closing this PR for now.

**Pull request closed without merging**